### PR TITLE
Feat/markdown image size 70

### PR DESCRIPTION
**Pull request by frontexpert**
This is solution for the issue #70 
I've customized the gatsby-remark-images plugin to resize images in markdown.

You can append "=`WIDTH`x`HEIGHT`" after image title text.
ex:
``
You can skip the `HEIGHT`
``

### CODE REVIEW

**Comment by frontexpert** [comment]
@shcheklein Please check my result and let me know your feedback.
Thanks

**Comment by frontexpert** [comment]
@shcheklein I just got your message.
> looks like somewhat an overkill to bring the whole plugin to us

Then, I am willing to upload my custom plugin to npm in public.
After that, I can update package.json file. so we can install the customized plugin(remark-images) instead of the original *gatsby-remark-images* plugin.

What about this?
Is this still not good solution?

Actually, Gatsby don't have any plugin for this solution now.
https://github.com/gatsbyjs/gatsby/issues/2609
This is why I customized the gatsby-remark-images now.

**Comment by shcheklein** [comment]
@frontexpert the problem with this approach is that we start depending on a third party plugin for a some core functionality. We'll need to support it on our own, keep it updated, etc. That's why this an overkill for this task.

**Comment by frontexpert** [comment]
OK, I got it. Thanks for your inform. Let me try again.

**Comment by frontexpert** [comment]
@shcheklein 
Please check my result again.

**Comment by shcheklein** [comment]
@frontexpert no, it's still not acceptable as a solution. It's still an overkill here. And we don't want to replace a regular plugin with our own version here.

If you don't see any other options in terms of implementing this custom markup, let's try to make it avoid stretching images (if it's 400x400 it should show it as 200x200 in browser, no stretching should happen). It might be enough to start. This way we will be able at least to pick the original size of the image.

**Comment by frontexpert** [comment]
@shcheklein I did not find another solution to implementing this custom markup. 

>  let's try to make it avoid stretching images (if it's 400x400 it should show it as 200x200 in browser, no stretching should happen). It might be enough to start. This way we will be able at least to pick the original size of the image.

Can you explain it more, please

**Comment by shcheklein** [comment]
@frontexpert custom plugin complexity should be reasonable. `external link` plugs nicely, it does not replace anything and it's pretty small and simple in terms of implementation. Similar approach if it works is acceptable. Replacing the whole `image-gatsby` is probably not.
